services/consensusalgo/benchmarkconsensus/test/harness.go

```diff
@@ -71,3 +76,82 @@ func (h *harness) createService(ctx context.Context) {
 		h.config,
 	)
 }
+
+func nonLeaderPublicKey() primitives.Ed25519Pkey {
+	return []byte{0x99}
+}
+
+func (h *harness) expectHandlerRegistrations() {
+	h.gossip.Reset().When("RegisterBenchmarkConsensusHandler", mock.Any).Return().Times(1)
+	h.blockStorage.Reset().When("RegisterConsensusBlocksHandler", mock.Any).Return().Times(1)
+}
+
+func (h *harness) verifyHandlerRegistrations(t *testing.T) {
+	ok, err := h.gossip.Verify()
+	if !ok {
+		t.Fatal("Did not register with Gossip:", err)
+	}
+	ok, err = h.blockStorage.Verify()
+	if !ok {
+		t.Fatal("Did not register with BlockStorage:", err)
+	}
+}
+
+func (h *harness) expectBlockCreation() {
+	h.consensusContext.Reset().When("RequestNewTransactionsBlock", mock.Any).Return(nil, nil).AtLeast(1)
+}
+
+func (h *harness) verifyBlockCreation(t *testing.T) {
```

Comment:
This doesn't, strictly, verify that blocks where created. It verifies that eventually all expectations from the mock were met. 

Please consider renaming this as well as the counterpart `verifyNoBlockCreation`

Reply:
I'll rename both